routes/incomeRoutes.js

```diff
@@ -1,11 +1,18 @@
 const express = require('express');
 const router = express.Router();
 const incomeController = require('../controllers/incomeController');
+const authMiddleware = require('../middleware/authMiddleware');
 
 // Endpoint do dodawania nowego przychodu
 router.post('/income', incomeController.addIncome);
 
 // Endpoint do pobierania przychodów
 router.get('/income', incomeController.getIncomes);
 
```

Comment:
puki co nie dodawaj autentykacji bo nie mamy kodu scalone, dodaj w opisie żeby to zaimplementować jak user będzie gotowy bo tak to oba routy nie działają
bez autentykacji oba robią co mają robić